lib/mhv_ac/registration_form.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+require 'common/models/form'
+
+module MHVAC
+  class RegistrationForm < Common::Form
+    attribute :icn, String
```

Comment:
@jcmeloni Can you confirm or disconfirm Kam's statement about LOA3? Because that's different than my understanding. I believe I as a joe random user have an LOA3 account because I have identity-proofed, but I'm not in MVI and don't have access to MHV-related features. I think level of assurance is orthogonal to MHV access.